frontend/src/app/add/page.tsx

```diff
@@ -84,9 +78,15 @@ const Add = () => {
     }, [handleErrors]);
 
     useEffect(() => {
-        void fetchCompanies();
         void fetchFormOptions();
-    }, [fetchCompanies, fetchFormOptions]);
+    }, [fetchFormOptions]);
+
+    useEffect(() => {
+        if (fetchCompanyError) {
+            console.log(fetchCompanyError);
```

Comment:
Same. Remove we don't want to console log this to the client. The handleError function should throw the error instead